### ISSUE TITLE
special chars are quated and thus not displayed correctly

### ISSUE THREAD

**Issue by SvenMeyer** [OpenZeppelin/solidity-docgen#376]
Example :+1: 

``` solidity
mapping(address => uint256) public addressMintedBalance;
```

Result:

### addressMintedBalance

```solidity
mapping(address &#x3D;&gt; uint256) addressMintedBalance
```

**Comment by frangio** [comment]
Does this render incorrectly when the Markdown is rendered?

Should this be unescaped because it's inside a block of code?

https://github.com/OpenZeppelin/solidity-docgen/blob/d7c993b2a9cee46eff4d24e61a76a822f864b691/src/themes/markdown/common.hbs#L4-L6

**Comment by frangio** [comment]
Fixed in cd714f5d0f48f934b5b03bcdea35dcb8f4a0cbac.
